src/App.css

```diff
@@ -169,17 +169,6 @@ a.text-link {
 
 .context-breadcrumbs {
   float: left;
-  margin-bottom: -1px;
-}
-
-.context-breadcrumbs .superscript-container {
-  position: relative;
-  left: -2px;
-  top: -3px;
-}
-
-.thought-annotation .context-breadcrumbs {
-  margin-right: 2px;
```

Comment:
The superscript positioning got a little messed up. Here is what I am seeing on Chrome OSX:

Before:
<img width="137" alt="Screen Shot 2019-11-28 at 8 03 33 PM" src="https://user-images.githubusercontent.com/750276/69840910-6881ad00-121a-11ea-8739-6471d6e1fbf7.png">

After:
<img width="137" alt="Screen Shot 2019-11-28 at 8 03 56 PM" src="https://user-images.githubusercontent.com/750276/69840916-6e778e00-121a-11ea-8da5-2d6b40f277fc.png">

If these CSS rules were causing problems in a different browser, could you please override them with browser-specific selectors rather than deleting? Thanks!

Reply:
It seems while a was editing some code went ahead touched. But it was unexpected. No need worries this was not the scope.

Reply:
The diff still shows that these lines are being deleted. Can you restore them to their originals? I won't be able to merge until this is corrected. Thanks!